kci_rootfs

```diff
@@ -62,16 +62,22 @@ class cmd_list_configs(Command):
 class cmd_build(Command):
     help = "Build rootfs image"
     args = [Args.config, Args.data_path]
+    opt_args = [Args.arch]
 
     def __call__(self, config_data, args, **kwargs):
         data_path = args.data_path
         config_name = args.config
+        arch = args.arch
         if config_name not in config_data['rootfs_configs']:
             print("{} invalid. Check 'kci_rootfs list_configs' for valid \
                     entries".format(config_name))
             return False
         config = config_data['rootfs_configs'][args.config]
-        kernelci.rootfs.build(config_name, config, data_path)
+        if arch is not None:
```

Comment:
I still think that `--arch` should be a required argument to run `kci_rootfs build --config=config-name --arch=arch-name` and always build only one rootfs per command line call.

Reply:
If we make `--arch` mandatory argument, do we still need to keep  `arch_list` inside `rootfs-configs.yaml`?

Reply:
Yes, these are different things.  The `rootfs-configs.yaml` file contains the list of architectures that should be built for each rootfs type.  Then, each architecture name can be passed in a separate call to `kci_rootfs build`.  I guess what is missing is the list of rootfs and arch combinations, and that's what `kci_rootfs list_variants` would be about.

Reply:
updated the patch. Will send a new PR for `list_variants` option.